Qt/mainwindow.cpp

```diff
@@ -293,28 +293,33 @@ void MainWindow::consoleAct()
 void MainWindow::raiseTopMost()
 {
 	setWindowState( (windowState() & ~Qt::WindowMinimized) | Qt::WindowActive);
-	raise();  
+	raise();
 	activateWindow();
 }
 
 void MainWindow::SetFullScreen(bool fullscreen) {
 	if (fullscreen) {
+#if !PPSSPP_PLATFORM(MAC)
 		menuBar()->hide();
-		
+
 		emugl->setFixedSize(QWIDGETSIZE_MAX, QWIDGETSIZE_MAX);
+		// TODO: Shouldn't this be physicalSize()?
 		emugl->resizeGL(emugl->size().width(), emugl->size().height());
 		// TODO: Won't showFullScreen do this for us?
 		setMaximumSize(QWIDGETSIZE_MAX, QWIDGETSIZE_MAX);
 		setFixedSize(QWIDGETSIZE_MAX, QWIDGETSIZE_MAX);
+#endif
```

Comment:
It seems that resizing the main window will trigger resizeGL automatically. This code should be unnecessary on Windows too.